examples/custom-config-ssl/configs/postgres-ha.yaml

```diff
@@ -0,0 +1,19 @@
+---
+bootstrap:
+  dcs:
+    postgresql:
+      parameters:
+        log_directory: pglogs
+        log_min_duration_statement: 0
+        log_statement: none
+        max_wal_senders: 6
+        shared_preload_libraries: pgaudit.so
+postgresql:  
+  pg_hba:
+    - local all postgres peer
+    - local all crunchyadm peer
+    - hostssl replication primaryuser 0.0.0.0/0 cert clientcert=1
```

Comment:
As of PostgreSQL 12, `clientcert` is now a loaded parameter.

In PostgreSQL 12, `clientcert` now accepts two values: `client=verify-ca` and `clientcert=verify-full`. This are similar to the `sslmode` versions of the parameter, i.e. 

- `verify-ca`: The PostgreSQL server will verify that the certificate that the client presents was signed by certificate in the trusted CA chain
- `verify-full`: Like `verify-ca`, but the additional check that the `CN` in the certificate matches the username (which could be aided by a map file). This is also pretty cool :smile:

`clientcert=1` is equivalent to `clientcert=verify-ca` in PostgreSQL 12...I believe `clientcert=1` is still honored in PostgreSQL 12, but would have to check.

Reply:
The PG 12 source confirms that `clientcert=1` is equivalent to `clientcert=verify-ca`:

https://github.com/postgres/postgres/blob/b4507a22fed9c2398895d1d4b94815ad1a053bb0/src/backend/libpq/hba.c#L1709-L1713